cookie-manager.js

```diff
@@ -0,0 +1,228 @@
+"use strict";
+
+(function () {
+
+    const init = function () {
+        readConfiguration();
+    };
+
+    const readConfiguration = function() {
+
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', '/cookie-manager.json');
+
+        xhr.onreadystatechange = function() {
+            if (xhr.readyState !== 4) {
+                return;
+            }
+            if (xhr.status !== 200) {
+                console.error('HTTP Error whilst requesting "/cookie-manager.json".', xhr.status, xhr.statusText);
```

Comment:
If we are using this as error diagnosis in production - look at wrapping in `if (typeof console == "undefined") { .. `to support only browser clients

Reply:
Nice catch. Services using this library will still be on older browsers.